Decred Wallet/Features/Security/Security.storyboard

```diff
@@ -1,96 +1,101 @@
-                            <constraint firstItem="dho-lD-rH7" firstAttribute="bottom" secondItem="53V-mQ-ZCO" secondAttribute="bottom" id="uXh-UQ-kUN"/>
-                            <constraint firstItem="dho-lD-rH7" firstAttribute="trailing" secondItem="53V-mQ-ZCO" secondAttribute="trailing" id="xEC-bP-GaI"/>
+                            <constraint firstItem="Yl8-6v-mJ9" firstAttribute="top" secondItem="pw2-ld-DVo" secondAttribute="bottom" id="0MZ-QI-6es"/>
+                            <constraint firstItem="pw2-ld-DVo" firstAttribute="leading" secondItem="dho-lD-rH7" secondAttribute="leading" id="5Gj-FJ-nB6"/>
+                            <constraint firstAttribute="bottomMargin" secondItem="Yl8-6v-mJ9" secondAttribute="bottom" constant="-1" id="Hvq-WT-smj"/>
+                            <constraint firstItem="pw2-ld-DVo" firstAttribute="top" secondItem="AyE-8L-BDu" secondAttribute="bottom" constant="8" id="OIi-Rm-dAh"/>
+                            <constraint firstItem="dho-lD-rH7" firstAttribute="trailing" secondItem="AyE-8L-BDu" secondAttribute="trailing" constant="24" id="Rfu-mY-5s4"/>
+                            <constraint firstItem="dho-lD-rH7" firstAttribute="trailing" secondItem="pw2-ld-DVo" secondAttribute="trailing" id="UTG-vj-C9o"/>
+                            <constraint firstItem="Yl8-6v-mJ9" firstAttribute="leading" secondItem="pw2-ld-DVo" secondAttribute="leading" id="eYQ-eH-KAD"/>
+                            <constraint firstItem="Yl8-6v-mJ9" firstAttribute="trailing" secondItem="pw2-ld-DVo" secondAttribute="trailing" id="g7F-J7-LbV"/>
+                            <constraint firstItem="AyE-8L-BDu" firstAttribute="leading" secondItem="dho-lD-rH7" secondAttribute="leading" constant="24" id="l1F-o8-aNX"/>
+                            <constraint firstItem="AyE-8L-BDu" firstAttribute="top" secondItem="dho-lD-rH7" secondAttribute="top" constant="24" id="vQQ-vX-af8"/>
                         </constraints>
                         <viewLayoutGuide key="safeArea" id="dho-lD-rH7"/>
                     </view>
                     <connections>
                         <outlet property="btnPassword" destination="FZP-4b-65P" id="k70-GQ-K7X"/>
                         <outlet property="btnPin" destination="b4m-te-ci6" id="nNQ-qG-I0q"/>
+                        <outlet property="securityPromptLabel" destination="AyE-8L-BDu" id="ukO-gA-dFm"/>
                     </connections>
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="Dvd-1h-Kar" userLabel="First Responder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="4626" y="-457"/>
+            <point key="canvasLocation" x="4624.6376811594209" y="-457.36607142857139"/>
         </scene>
         <!--Tab Bar Controller-->
         <scene sceneID="KAG-yl-ivD">
```

Comment:
This view presentation should not be fullscreen. check the height, it should take around 0.6(multiplier) of the screen from bottom...put your components in a container view and make the base view transparent so the view Controller presenting it is visible.

Reply:
This should be fixed in a different PR